examples/src/main/content/jcr_root/etc/groovyconsole/scripts/aecu/aecu-examples/project3/script1.groovy

```diff
@@ -0,0 +1,10 @@
+println aecu
```

Comment:
File should be removed. We should not change we.retail in our example package.
Alternative: use dry run